content/en/docs/tutorials/cluster-management/kubelet-standalone.md

```diff
@@ -0,0 +1,526 @@
+* Learn how the different components of the solution interact among themselves.
+
+{{< caution >}}
+The Kubelet configuration used for this tutorial is insecure by design and should
+_not_ be used in a production environment.
+{{< /caution >}}
+
+## {{% heading "prerequisites" %}}
+
+- Admin (`root`) access to a Linux system that uses `systemd` and `iptables`
+  (or nftables with `iptables` emulation).
+- Access to the Internet to download the components needed for the tutorial, such as:
+    - A {{< glossary_tooltip text="container runtime" term_id="container-runtime" >}}
+    that implements the Kubernetes {{< glossary_tooltip term_id="cri" text="(CRI)">}}.
+    - Network plugins (these are often known as
+    {{< glossary_tooltip text="Container Networking Interface (CNI)" term_id="cni" >}})
+- Required CLI tools: `curl`, `tar`, `jq`.
+
+<!-- lessoncontent -->
+
+## Prepare the system
+
+### Swap configuration
+
+The default behavior of kubelet is to fail to start if swap memory is detected on a node.
+This means that swap should either be disabled or tolerated by kubelet.
+
+{{< note >}}
+If you configure the kubelet to tolerate swap, the kubelet still configures Pods (and the
+containers in those Pods) not to use swap space. To find out how Pods can actually
+use the available swap, you can read more about
+[swap memory management](/docs/concepts/architecture/nodes/#swap-memory) on Linux nodes.
+{{< /note >}}
+
+If you have swap memory enabled, either disable it or add `failSwapOn: false` to the
+kubelet configuration file.
+
+To check if swap is enabled:
+
+```shell
+sudo swapon --show
+```
+
+If there is no output from the command, then swap memory is already disabled.
+
+To disable swap temporarily:
+
+```shell
+sudo swapoff -a
+```
+
+To make this change persistent across reboots:
+
+Make sure swap is disabled in either `/etc/fstab` or `systemd.swap`, depending how it was
+configured on your system.
+
+### Enable IPv4 packet forwarding
+
+To check if IPv4 packet forwarding is enabled:
+
+```shell
+cat /proc/sys/net/ipv4/ip_forward
+```
+
+If the output is `1`, it is already enabled. If the output is `0`, then follow next steps.
+
+To enable IPv4 packet forwarding, create a configuration file that sets the
+`net.ipv4.ip_forward` parameter to `1`:
+
+```shell
+sudo tee /etc/sysctl.d/k8s.conf <<EOF
+net.ipv4.ip_forward = 1
+EOF
+```
+Apply the changes to the system:
+
+```shell
+sudo sysctl --system
+```
+
+The output is similar to:
+
+```
+...
+* Applying /etc/sysctl.d/k8s.conf ...
+net.ipv4.ip_forward = 1
+* Applying /etc/sysctl.conf ...
+```
+
+## Download, install, and configure the components
+
+{{% thirdparty-content %}}
+
+### Create the sources directory
+
+Download the latest available versions of the required packages (recommended).
+
+The relative path to the downloaded software is assumed to be the local `sources` directory.
+
+```shell
+mkdir sources
+cd sources
+```
+
```

Comment:
I don't think this is relevant to the tutorial.

Reply:
Agree. Done